### PR TITLE
[add]: hover-on-add-transaction-btn

### LINKED ISSUE #11: Add hover property on Button
You can see the following botton img

image

### PR DESCRIPTION
**Pull request by PratikMbade**
## Fix
fixes #11 

## Type of PR
Add Feature

## What does this PR do
It adds hover on the "Add Transaction" button

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -113,8 +113,33 @@ input[type='number'] {
...
+  width: 0;
+  height: 100%;
+  mix-blend-mode: lighten;
+  background: #eb8201;
+  transition: all 0.5s  cubic-bezier(0.420, 0.000, 0.580, 1.000);;
+}
+
+.btn:hover{
+    background:  #f52525;
+  }
```
Comment (shivam-sharma7) [inline src/App.css:134]:
```suggestion
.btn:hover{
    background:  #31D111;
  }
```

Comment (shivam-sharma7) [reply src/App.css:134]: can you add this color

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (PratikMbade) [reply src/App.css:134]: image

bro your chosen color is looks great

Comment (PratikMbade) [reply src/App.css:134]: I've done with the changes I will do another pull request. wait for it

Comment (shivam-sharma7) [reply src/App.css:134]: you can do same in this PR

Comment (PratikMbade) [reply src/App.css:134]: @shivam-sharma7  can please guide it on how to do it in the same PR

Comment (PratikMbade) [reply src/App.css]: It shows me :- remote: Permission to shivam-sharma7/Expense-Tracker.git denied to PratikMbade.
fatal: unable to access 'https://github.com/shivam-sharma7/Expense-Tracker/': The requested URL returned error: 403

Comment (PratikMbade) [reply src/App.css]: I did another commit. just check it

**Review by shivam-sharma7 — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**